src/lib.rs

```diff
@@ -508,11 +508,18 @@ macro_rules! __impl_bitflags {
             )+
 
             /// Returns an empty set of flags.
+            #[cfg(not(feature = "const_empty"))]
             #[inline]
             pub fn empty() -> $BitFlags {
                 $BitFlags { bits: 0 }
             }
 
+            #[cfg(feature = "const_empty")]
```

Comment:
I think this does something other than what you intended. This attribute ends up in the output of the bitflags macro so it would look for a `const_empty` feature in the caller's crate, not the `const_empty` feature you added above.

Reply:
_Sigh_. You're right. This is easily worked around by defining the method in your own codebase. In light of that, I think I'm just going to close this, and send a PR once `const fn` is stabilized.